src/lammpsio/snapshot.py

```diff
@@ -106,7 +106,14 @@ def from_hoomd_gsd(cls, frame):
 
         return snap, type_map
 
-    def to_hoomd_gsd(self, type_map=None):
+    def to_hoomd_gsd(
+        self,
+        type_map=None,
```

Comment:
If we stay with this, we should note to consider renaming`type_map` to `particle_type_map` just to be clear.

Reply:
This will be an issue for `from_hoomd_gsd` as well since that returns a `type_map`. We could do like you've done here. We could also accept a dict of dicts (keyed on "particles", "bonds", etc.), and for backwards compatibility, convert a regular dict to the particles key. Or, we could create a typemap object that stores all of these, and require this be passed when mapping topology data (and internally convert the dict of particle types to this object).